CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [Unreleased]
 
+- [`named`]: ignore Flow `typeof` imports and `type` exports ([#1345], thanks [@loganfsmyth])
```

Comment:
you’ll need an explicit link for the pr number (see the bottom) and one for your name if you haven’t contributed previously

Reply:
Woops, fixed! Copy-pasted and just assumed it was some feature of Github linking I didn't know about hahah.